common-lib/src/main/java/com/google/cloud/tools/intellij/BasePluginInfoService.java

```diff
@@ -41,6 +41,7 @@
   private final String userAgent;
   private final IdeaPluginDescriptor plugin;
   private final FlagReader flagReader;
+  private static final String PLUGIN_NAME_EXTERNAL = "gcloud-intellij";
```

Comment:
@etanshaul I had a discussion with @patflynn. We will do some overhaul about this thing, so we don't need to think about this much in this PR. Let's leave this field here for now.